doc/generic/pgf/pgfmanual-en-base-matrices.tex

```diff
@@ -191,9 +187,9 @@ \subsection{The Matrix Command}
     wish to be aware of:
     %
     \begin{itemize}
-        \item It is not necessary to actually mention |\pgfmatrixnextcell| or
+        \item It is not necessary to actually mention |&| or
             |\pgfmatrixendrow| inside the \meta{matrix cells}. It suffices that
-            the  macros inside \meta{matrix cells} expand to these macros
+            the  macros inside \meta{matrix cells} expand to these
```

Comment:
now that the row is changed anyway, we can directly remove one of the remaining "double space" instances :)

```suggestion
            the macros inside \meta{matrix cells} expand to these
```